yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -508,8 +508,7 @@ def run(self, information):
         if acodec != 'copy':
             more_opts = self._quality_args(acodec)
 
-        # not os.path.splitext, since the latter does not work on unicode in all setups
-        temp_path = new_path = f'{path.rpartition(".")[0]}.{extension}'
+        temp_path = new_path = f'{os.path.splitext(path)[0]}.{extension}'
```

Comment:
Better to use `replace_extension` with `expected_real_ext=information['ext']`

Reply:
Thanks for the patience, just updated.